sources/packages/web/src/components/students/applicationTracker/ApplicationProgressBar.vue

```diff
@@ -63,9 +63,21 @@ import Assessment from "@/components/students/applicationTracker/Assessment.vue"
 import Enrolment from "@/components/students/applicationTracker/Enrolment.vue";
 import Completed from "@/components/students/applicationTracker/Completed.vue";
 
-interface ApplicationEndStatusIconDetails {
-  endStatusType?: "success" | "warning" | "error";
-  endStatusIcon?: string;
+class StatusIconDetails {
```

Comment:
why a class? why did we go for the method in the setup?

Reply:
I believe that after the chat it was clear, either was I am changing it to avoid confusion in the future.